.github/workflows/ci-actions.yml

```diff
@@ -512,14 +577,13 @@ jobs:
           CATEGORY: ${{matrix.category}}
         run: |
           for i in $TEST_MODULES
-          do modules+=("integration-tests/$i"); done
-          IFS=,
-          eval mvn -pl "${modules[*]}" $NATIVE_TEST_MAVEN_OPTS
+          do modules+="integration-tests/$i,"; done
```

Comment:
Additionally to `eval` (see futher up) I also had issues with `IFS=,`.
Since Maven doesn't mind a trailing `,`, I simplified it all a bit.